content/manual/running/hosting.md

```diff
@@ -7,196 +7,491 @@ weight = 2
-[do nginx install]: https://www.digitalocean.com/community/tutorials/how-to-install-nginx-on-ubuntu-18-04
-[do nginx config]: https://www.digitalocean.com/community/tutorials/how-to-deploy-a-go-web-application-using-nginx-on-ubuntu-18-04
-[do ssl config]: https://www.digitalocean.com/community/tutorials/how-to-secure-nginx-with-let-s-encrypt-on-ubuntu-18-04
-[urbit basic cloud install]: https://medium.com/@urbitlive/hello-world-urbit-edition-install-boot-and-run-your-urbit-planet-on-a-10-cloud-server-b9579745b9a8
-[do initial setup]: https://www.digitalocean.com/community/tutorials/initial-server-setup-with-ubuntu-18-04
-[blog github]: https://github.com/zalberico/zalberico.github.io
-
-If you plan to use Nginx as a reverse proxy for your Urbit, it is important that you include the following settings in your configuration in order to allow the Landscape web client to properly communicate with your Urbit:
-` chunked_transfer_encoding off; proxy_buffering off; proxy_cache off; `
+The goal of this guide is to have clear and easy to follow best practices for
+deploying an Urbit node to a server you control in the cloud. Deploying in the
+cloud allows you to access your Urbit from any device.
+
+Most Urbit users start out running their ship locally on one machine in order to
+play with it, but this means when your machine is offline your Urbit node is
+offline too (and can't get updates). You can also only access your Urbit from
+that one machine.
+
+This guide uses Digital Ocean as the cloud provider, but others can be used. If
+using another provider, the setup script provided and other server configuration
+instructions may need to be modified or done manually.
+
+## 1. Create a Droplet
+
+Create an account on [Digital Ocean](https://digitalocean.com). Once you make an
+account, choose "Deploy a virtual machine".
+
+You should see the page below where you can create your Droplet, aka Virtual Machine:
+
+![do screenshot](https://media.urbit.org/operators/manual/running/hosting/do-screenshot.png)
+
+Fill out the options like so:
+
+#### Image
+
+Ubuntu 22.04 x64
+
+#### Plan
+
+- Shared CPU: Basic
+- CPU options: Regular with SSD
+- 2GB / 1 CPU ($12/mo)
+
+You can choose a more powerful option if you'd like but the $12 option should be
+sufficient. Note Urbit needs 2GB of memory; it's possible to choose a cheaper
+option and run it with less memory by using swap but it will impact performance.
+
+#### Add block storage
+
+The $12 plan includes 50GB which should be sufficient for quite some time, so
+you can skip this.
+
+#### Datacenter region
+
+Choose the region closest to you.
+
+#### VPC Network
+
+Leave this as default.
+
+#### Authentication
+
+In the "Authentication" field, select "SSH keys" and hit "New SSH Key". Run the
+following command in the terminal on your local machine, replacing
+`riclen-tinlyr` with the name of your ship (sans the leading `~`):
```

Comment:
significant?

Reply:
how do you mean?

Reply:
The ship name, as opposed to ~sampel-palnet.

Reply:
Just curious.

Reply:
bash expands leading tildes to a few different variables so you'd have to refer to the file with quotes around it which would be a bit annoying